ALUM/ALUM/Views/SessionDetailsView/SessionDetailsScreen.swift

```diff
@@ -0,0 +1,425 @@
+//
```

Comment:
@HarshGurnani I combined Mentee and Mentor Details so that it was easier to route